menas/src/main/scala/za/co/absa/enceladus/menas/controllers/LandingPageController.scala

```diff
@@ -48,4 +59,14 @@ class LandingPageController @Autowired() (datasetRepository: DatasetMongoReposit
       todaysStats <- runsService.getTodaysRunsStatistics()
     } yield LandingPageInformation(dsCount, mtCount, schemaCount, runCount, todaysStats)
   }
+
+  @Async
+  @Scheduled(initialDelay = 1000, fixedDelay = 900000)
```

Comment:
Also, is 15 minutes refresh a good frequency?

Reply:
Good question, these annotations also expect constants so I couldn't find a quick way to make them configurable

Reply:
As written elsewhere, I hope this is a quickfix. So let's not dwell on it for too long. 
On second thought, 15 minutes or 5 doesn't make much difference :-)

Reply:
Hmm maybe 5 mins is more reasonable from user perspective.